### PR TITLE
Update afk.go.tmpl

### PR DESCRIPTION
**Pull request by DaviiD1337**

### CODE REVIEW

File: afk/afk.go.tmpl [hunk 1]
```diff
@@ -11,16 +11,15 @@
...
 {{/* CONFIGURATION VALUE ENDS HERE */}}
 
-{{/* Dont edit below if you don't know what you're doing ! */}}
 {{ $time := 0 }}{{ $afk := dbGet .User.ID "afk" }}{{ $prefix := index (reFindAllSubmatches `.*?: \x60(.*)\x60\z` (execAdmin "Prefix")) 0 1 }}
 {{ if $args := .Args }}
     {{ if eq (lower (index $args 0)) (print $prefix "afk") }}
         {{ if and (ge (len $args) 2) (not $afk) }}
             {{ $args = (slice .Args 1) }}
-            {{ $message := "" }}
+            {{ $message := "No reason setted" }}
```
Comment (Olde7325) [inline afk/afk.go.tmpl:22]: A more sensible afk reason when none is set:
```suggestion
            {{ $message := "afk" }}
```

File: afk/afk.go.tmpl [hunk 2]
```diff
@@ -43,7 +42,9 @@
             {{ with and $duration (toDuration $duration) }} {{ $parsedDur = . }} {{ end }}
             {{ if $parsedDur }}
                 {{ dbSetExpire .User.ID "afk" $message (toInt $parsedDur.Seconds) }}
-            {{ else }} {{ dbSet .User.ID "afk" $message }} {{ end }}
+            {{ else }} 
+                {{ dbSet .User.ID "afk" $message }} {
+            {{ end }}
```
Comment (Olde7325) [inline afk/afk.go.tmpl:47]: This doesn't look right
```suggestion
            {{ else }} 
                {{ dbSet .User.ID "afk" $message }}
            {{ end }}
```

File: afk/afk.go.tmpl [hunk 3]
```diff
@@ -11,16 +11,15 @@
 */}}
 
 {{/* CONFIGURATION VALUE STARTS HERE */}}
-{{ $removeAfkOnMessage := true }}{{/* Remove the AFK database from an user if he/she types something in chat. */}}
+{{ $removeAfkOnMessage := false }}{{/* Remove the AFK database from an user if he/she types something in chat. */}}
```
Comment (Olde7325) [inline afk/afk.go.tmpl:14]: I would let this be the default setting:
```suggestion
{{ $removeAfkOnMessage := true}}{{/* Remove the AFK database from an user if he/she types something in chat. */}}
```

**Review by Olde7325 — CHANGES_REQUESTED** [review]
Looking good so far, just a few things.

File: afk/afk.go.tmpl [hunk 4]
```diff
@@ -61,7 +62,7 @@
             {{ if gt .ExpiresAt.Unix 0 }} {{ $eta = humanizeDurationSeconds (.ExpiresAt.Sub currentTime) | printf "*%s will be back in around %s.*" $user.Username }} {{ end }}
             {{ sendMessage nil (cembed
                 "author" (sdict "name" (printf "%s is AFK" $user.String) "icon_url" ($user.AvatarURL "256"))
-                "description" (joinStr "\n\n" $eta .Value)
+                "description" (print $eta "\n\nReason: **" .Value "**")
```
Comment (jo3-l2) [inline afk/afk.go.tmpl:65]: Honestly not a super big fan of this change, but if others like it better than the original, that's okay.
However, I don't think we should be providing a default message unless strictly necessary. Here's how I would format the description:

* If there is both an ETA and reason, prefer `<ETA>\n<REASON>`
* If there is only an ETA, prefer `<ETA>`
* If there is only a reason, prefer `<REASON>`
* If there is neither an ETA nor a reason, prefer `*No reason provided*`

File: afk/afk.go.tmpl (hunk 1 above)
Comment (jo3-l2) [reply afk/afk.go.tmpl:22]: I don't think we should be using a default unless strictly necessary. See review comment later on.

File: afk/afk.go.tmpl (hunk 3 above)
Comment (jo3-l2) [reply afk/afk.go.tmpl:14]: Yes, reverting this change would be preferable. From feedback on the first version of AFK, many users were used to the AFK clearing when they sent a message.

**Review by jo3-l2 — CHANGES_REQUESTED** [review]

File: afk/afk.go.tmpl (hunk 3 above)
Comment (jo3-l2) [reply afk/afk.go.tmpl:14]: Agreed.

File: afk/afk.go.tmpl (hunk 1 above)
Comment (jo3-l2) [reply afk/afk.go.tmpl:22]: I think we shouldn't have a default message at all unless absolutely necessary, see my last review comment.

File: afk/afk.go.tmpl (hunk 4 above)
Comment (Olde7325) [reply afk/afk.go.tmpl:65]: Fair enough, couldn't made any sense out of the abbrevation "ETA", thus not commenting on this change.

File: afk/afk.go.tmpl (hunk 3 above)
Comment (jo3-l) [reply afk/afk.go.tmpl:12]: Why was this resolved without any changes?

**Comment by jo3-l** [comment]
Thanks!

**Pull request merged**